cmd/svcat/class/get_cmd.go

```diff
@@ -46,6 +48,7 @@ func NewGetCmd(cxt *command.Context) *cobra.Command {
   svcat get classes
   svcat get class mysqldb
   svcat get class --uuid 997b8372-8dac-40ac-ae65-758b4a5075a5
+  svcat get classes --broker
```

Comment:
This is not a valid call to svcat. The name of the broker is required when using the --broker flag.

Reply:
agreed